modules/uncertainty/R/ensemble.R

```diff
@@ -264,11 +264,11 @@ write.ensemble.configs <- function(defaults, ensemble.samples, settings, model,
     if (!is.null(con)){
       #-- lets first find out what tags are required for this model
       required_tags <- dplyr::tbl(con, 'models') %>%
-        dplyr::filter(id == !!as.numeric(settings$model$id)) %>%
+        dplyr::filter(.data$id == !!as.numeric(settings$model$id)) %>%
```

Comment:
@Its-Maniaco I missed earlier that this causes a [note fom R CMD check](https://github.com/PecanProject/pecan/runs/5826374009?check_suite_focus=true#step:6:1498). To resolve, please add the following between current lines 207 and 208 (GitHub won't let me suggest changes there directly, because it's not near a changed line):

```
##' @importFrom rlang .data
```

Reply:
@Its-Maniaco we finally got the Github Action working again on our side and your PR seems to be failing because of the issues noted above
```
  R check of modules/uncertainty returned new problems:
  checking R code for possible problems ... NOTE: write.ensemble.configs: no visible binding for global variable '.data'
  ```